docs/networkpolicy-recommendation.md

```diff
@@ -134,3 +140,26 @@ policies to a YAML file and apply it using `kubectl`:
 theia policy-recommendation retrieve e998433e-accb-4888-9fc8-06563f073e86 -f recommended_policies.yml
 kubectl apply -f recommended_policies.yml
 ```
+
+### List all policy recommendation jobs
+
+The `theia policy-recommendation list` command lists all policy recommendation
+jobs. `CreateTime`, `CompleteTime`, `ID` and `Status` of each policy
+recommendation job will be displayed in table format. For example:
+
+```bash
+> theia policy-recommendation list
+CreateTime          CompleteTime        ID                                   Status
+2022-06-17 18:33:15 N/A                 2cf13427-cbe5-454c-b9d3-e1124af7baa2 RUNNING
+2022-06-17 18:06:56 2022-06-17 18:08:37 e998433e-accb-4888-9fc8-06563f073e86 COMPLETED
+```
+
+### Delete a policy recommendation job
+
+The `theia policy-recommendation delete` command is used to delete a policy
+recommendation job. To delete the policy recommendation job created above, run:
```

Comment:
I notice there is a rare case that if users try to delete a job before the driver pod is created, the job may still keep running till completion. And users will receive a message saying the job is successfully deleted. Not sure if we should deal with this case by returning another message or doc it here.

Reply:
Thanks Yanjun, I have seen this corner case too during e2e test. In that case, the `sparkapplication` cr has been deleted successfully but the deriver/exec pods will still be created later, maybe I should create an issue in the spark operator repo to hear their solutions.